spec: add support for SSE, FWFT, DBTR, and MPXY extensions (#120)
rustsbi/rustsbi: library/sbi-rt/CHANGELOG.md

- Add C language naming alias tags to all functions of the sbi-rt library
- rt: add DBTR extension support to SBI implementation.
- dbtr: use `TriggerMask` structure in sbi-rt DBTR functions

> - rt: add SSE extension support to SBI implementation.
> - rt: add MPXY extension support to SBI implementation.

Reviewer: ```suggestion
- rt: add structure for SSE, FWFT, DBTR, and MPXY extensions
```

Add one changelog entry only.

[0.0.3]: https://github.com/rustsbi/sbi-rt/compare/v0.0.2...v0.0.3
[0.0.2]: https://github.com/rustsbi/sbi-rt/compare/v0.0.1...v0.0.2
[0.0.1]: https://github.com/rustsbi/sbi-rt/releases/tag/v0.0.1

> ### Related Issues
> Closes #118.
> ```

Reviewer: ```suggestion
```
Do not include extra text in changelog.
